frame/support/procedural/src/pallet/expand/call.rs

```diff
@@ -89,7 +107,27 @@ pub fn expand_call(def: &mut Def) -> proc_macro2::TokenStream {
 		&docs[..]
 	};
 
+	let maybe_compile_error = if def.call.is_none() {
+		quote::quote!{
+			compile_error!("Pallet does not have #[pallet::call] defined. Did you forget to include it?");
```

Comment:
Can we name the pallet here?

More likely the response should be "Maybe you should remove `Call`" rather than "did you forget to include it"

Reply:
The latest changes have made it so that `#[pallet::call]` is entirely optional, so their pallet wouldn't be broken if they missed it out.

However I do agree that it's better to shift the perspective from the pallet author and into the runtime creator. I've reworded the compiler error message accordingly.